clickhouse-admin/tests/integration_test.rs

```diff
@@ -51,7 +51,7 @@ async fn test_lgif_parsing() -> anyhow::Result<()> {
         .context("failed to generate config")?;
     deployment.deploy().context("failed to deploy")?;
 
-    wait_for_keepers(&log, &deployment, vec![KeeperId(1)]).await?;
+    wait_for_keepers(&log, &deployment, vec![clickward::KeeperId(1)]).await?;
```

Comment:
Was this change to prevent conflicts with `clickhouse_admin_types::KeeperId` ?

Reply:
yep!